src/components/portfolio_elements/PortfolioPreview.vue

```diff
@@ -1,22 +1,33 @@
+<script setup lang="ts"></script>
+
 <template>
-  <div>
-    <img :src="imageSrc" alt="Description of the image" class="static-image" />
-  </div>
+  <li>
+    <img :src="getPreviewImageUrl(imageSrc)" alt="Description of the image" class="static-image" />
+  </li>
 </template>
 
-<script>
+<script lang="ts">
 export default {
   name: 'PortfolioPreview',
   props: {
     imageSrc: {
       type: String,
       required: true
     }
+  },
+  methods: {
+    getPreviewImageUrl(imagePath: string): string {
+      return new URL('/src/assets/images/portfolio/' + imagePath, import.meta.url).href
```

Comment:
Love this but let's move the path to an env variable or the like so you can maintain it outside of the component.

Reply:
For instance, you could define it as an alias in the Vite config.